dev/main.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+/* global requirejs */
+/* global define */
+
+requirejs.config({
+	paths: {
+		jquery: '//ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min',
```

Comment:
after chatting - looks like this is just here to check it's working alongside jquery rather than depending on it